JDBC/OAuthSampleApp/src/main/java/OAuthSampleApp.java

```diff
@@ -79,17 +79,17 @@ private static void SetUpDbForOAuth() throws Exception
 		String USER = prop.getProperty("User");
 		String CLIENT_ID = prop.getProperty("ClientId");
 		String CLIENT_SECRET = prop.getProperty("ClientSecret");
-		String INTROSPECT_URL = prop.getProperty("TokenUrl") + "introspect";
+		String DISCOVERY_URL = prop.getProperty("DiscoveryUrl");
```

Comment:
Is the setup working with the discovery URL but not the token URL?

Reply:
Yes, It's not working with the introspect URL.